src/acl_kernel_if.cpp

```diff
@@ -44,7 +44,8 @@ acl_process_autorun_profiler_scan_chain(unsigned int physical_device_id,
 // to verify that the hardware and HAL have an identical view of the CSR
 // address map.
 #define CSR_VERSION_ID_181 (3)
-#define CSR_VERSION_ID (4)
+#define CSR_VERSION_ID_20222 (4)
+#define CSR_VERSION_ID (5)
```

Comment:
We could introduce an alias for the latest version:
```suggestion
#define CSR_VERSION_ID_2022_3 (5)
#define CSR_VERSION_ID CSR_VERSION_ID_2022_3
```
The explicitly versioned constant may then be used below.